app/javascript/app/document-capture/hooks/use-csrf.js

```diff
@@ -0,0 +1,5 @@
+function useCSRF() {
+  return document.getElementsByName('csrf-token')[0].content;
```

Comment:
As written, this doesn't really seem to _need_ to be a React hook, since it doesn't manage its own state, nor does it handle side effects as a consequence of component lifecycle.

I could imagine a hook where it uses a new [Context](https://reactjs.org/docs/context.html) for the CSRF token, which — as we've done with other providers — would be assigned at the top level `packs/document-capture.jsx`, with the added advantage of keeping most of the integration (app-markup-specific) details out of the `app/document-capture` logic.

Reply:
> As written, this doesn't really seem to _need_ to be a React hook, since it doesn't manage its own state, nor does it handle side effects as a consequence of component lifecycle.
> 
> I could imagine a hook where it uses a new [Context](https://reactjs.org/docs/context.html) for the CSRF token, which — as we've done with other providers — would be assigned at the top level `packs/document-capture.jsx`, with the added advantage of keeping most of the integration (app-markup-specific) details out of the `app/document-capture` logic.

Updated in d6fda5f08fe1c9c56b343121e3c257513ce979db.